### PR TITLE
Form and fetch branch

### PR DESCRIPTION
**Pull request by Suprame4**
feat: add buttons that will redirect to form components that will utilize endpoints via fetch

### CODE REVIEW

**Review by VicBriar — CHANGES_REQUESTED** [review]
the buttons look great!
I have a question about the addForm, setAddForm state item. On App.js, lines 22-33, In the view state, there is a object called 'item draft' that I created to hold the values you've put in addForm. Personally, I prefer that we only have one state variable; it makes renders more efficient. However, I can be flexible on this. Either way, we do need to decide which state piece we are going to keep; having both is redundant.

File: public/react/components/App.js [hunk 1]
```diff
@@ -87,12 +131,38 @@ export const App = () => {
 				return(<>
 					<SingleItem item={view.item} />
 				</>);
+			//add item logic
```
Comment (JMantello) [inline public/react/components/App.js:134]: Are these new pages to be added?

Comment (Suprame4) [reply public/react/components/App.js:134]: Yeah, but only the buttons show so far.

**Comment by Suprame4** [comment]
>

**Comment by Suprame4** [comment]
> the buttons look great! I have a question about the addForm, setAddForm state item. On App.js, lines 22-33, In the view state, there is a object called 'item draft' that I created to hold the values you've put in addForm. Personally, I prefer that we only have one state variable; it makes renders more efficient. However, I can be flexible on this. Either way, we do need to decide which state piece we are going to keep; having both is redundant.

Is it okay if I comment out my logic for now, and use your code for the forms and see how it goes?

**Review by VicBriar — APPROVED** [review]
that's fine! You can comment out for sure. Thanks for taking a look at it!

**Pull request merged**